frontend/src/lib/components/VisibilitySensor/visibilitySensorLogic.tsx

```diff
@@ -39,10 +39,11 @@ export const visibilitySensorLogic = kea<visibilitySensorLogicType>({
         },
     }),
 
-    selectors: () => ({
+    selectors: ({ props }) => ({
         checkIsVisible: [
-            (selectors) => [selectors.innerHeight, (_, props) => props.offset || 0],
-            (windowHeight, offset) => (element: HTMLElement) => {
+            (selectors) => [selectors.innerHeight],
```

Comment:
Just for future reference, this is what borks offset props :). Passing as a fn allows updated props to reflect here.

Reply:
I'm curious why this happens though? I thought we cloud read props directly in a selector. Maybe @mariusandra you have some context?

Reply:
I'm very out of the loop here. Should I dig in to something?